private/buf/cmd/buf/command/generate/generate.go

```diff
@@ -275,7 +275,7 @@ func (f *flags) Bind(flagSet *pflag.FlagSet) {
 		&f.TypesDeprecated,
 		typeDeprecatedFlagName,
 		nil,
-		"The types (message, enum, service) that should be included in this image. When specified, the resulting image will only include descriptors to describe the requested types. Flag usage overrides buf.gen.yaml",
+		"The types (package, message, enum, service, method) that should be included in this image. When specified, the resulting image will only include descriptors to describe the requested types. Flag usage overrides buf.gen.yaml",
```

Comment:
Can you also update the description for `typeFlagName`?

Reply:
I pushed a change that includes this, and that also updates the flag of the same name (and same function) for `buf build`.